content/docs/user-guide/gems/reference/aws/aws-core/cdk-application.md

```diff
@@ -5,21 +5,28 @@ weight: 200
 toc: true
 ---
 
-The [Cloud Development Kit](https://docs.aws.amazon.com/cdk/latest/guide/home.html) (CDK) is a software development framework from AWS for defining cloud infrastructure for your project and provisioning it through **AWS CloudFormation**.
+The [Cloud Development Kit](https://docs.aws.amazon.com/cdk/v2/guide/home.html) (CDK) is a software development framework from AWS for defining cloud infrastructure for your project and provisioning it through **AWS CloudFormation**.
+
+{{< note >}}
+The AWS Gems have been updated to support CDK v2, the new long term CDK version. CDK v1 entered maintenance on June 1, 2022, and will now receive only critical bug fixes and security patches. 
+
+For those still using the CDK v1, a legacy snapshot of the sample applications are available in a `cdkv1` folder inside each Gem. See the [CDK migration Guide](https://docs.aws.amazon.com/cdk/v2/guide/migrating-v2.html) for advice about how to migrate legacy CDK applications to the CDK v2. 
+{{< /note >}}
 
 The AWS Core Gem includes an optional Python CDK application that provides two stacks:
 
 1. A core stack to use as the basis for a project's CDK application.
-1. An example stack with example resources that can be connected to **ScriptBehavior** samples in Core.
+2. An example stack with example resources that can be connected to **ScriptBehavior** samples in Core.
```

Comment:
There is no issue with renumbering this, but Hugo will automatically number lists that only use `1.` for each step.

Reply:
This was part of an automated cleanup, confirmed they do display directly but will try to avoid renumbering as it prevents future breakage when folks add new items.